src/qz/ws/PrintSocketClient.java

```diff
@@ -176,6 +176,19 @@ public void onMessage(Session session, Reader reader) throws IOException {
         }
     }
 
+    private JSONObject cleanupMessage(JSONObject msg) {
+        msg.remove("promise"); //never needed java side
```

Comment:
@bberenz if this is getting accidentally serialized, should we be cleaning this up JavaScript side too or is that too difficult?

Reply:
we use it js side, it is functions though which don't serialize so its always blank here

Reply:
So it turns out stringify supports a replacer function, so we can remove this from the API as well.

https://stackoverflow.com/questions/4910567/hide-certain-values-in-output-from-json-stringify
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#parameters

Reply:
@bberenz checking in on the `replacer` proposal to remove this from JS as well.